templates/taxi/car_detail.html

```diff
@@ -1,12 +1,21 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
-  </ul>
+  <h1>{{ car.manufacturer.name }}, {{ car.model }} ({{ car.manufacturer.country }})</h1>
+    <table class="table">
+    <tr>
+        <th>Driver</th>
+        <th></th>
+        <th></th>
+        <th></th>
+    </tr>
+      <tr>
+          {% for driver in car.drivers.all %}
+              <td>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</td><a href=""></a>
+              <td></td>
+              <td><a class="text-decoration-none" href="{% url "taxi:delete-car" car.id%}"><b>DELETE</b></a></td>
+              <td><a class="text-decoration-none" href="{% url "taxi:update-car" car.id%}"><b>UPDATE</b></a></td>
```

Comment:
You want to delete/update the car but the buttons are in one line with the driver, for the user these buttons look like for delete/update driver in car, but not the car itself, move it on top to the car name, or to the bottom(under the drivers list), to make it more user-friendly.